packages/react-devtools/src/devtools.tsx

```diff
@@ -157,15 +157,10 @@ export const TanStackDevtools = ({
                   },
             render: (e, theme) => {
               const id = e.getAttribute('id')!
-              const target = e.ownerDocument.getElementById(id)
-
-              if (target) {
-                setPluginContainers((prev) => ({
-                  ...prev,
-                  [id]: e,
-                }))
-              }
-
+              setPluginContainers((prev) => ({
```

Comment:
Because we are rendering the component in a shadow DOM node, `e.ownerDocument.getElementById(id)` always returns `null` and the plugins are never mounted. 

I'm not certain I understand what this `if` statement guards against since I thought the elements  were defined by the devtools plugin itself. Shouldn't they always exist?